.travis.yml

```diff
@@ -31,7 +30,9 @@ deploy:
   # Push image to Dockerhub on tag
   - provider: script
     skip_cleanup: true
-    script: bash scripts/docker-push.sh $IMAGE_NAME $TRAVIS_TAG $TRAVIS_CPU_ARCH
+    script:
+      - make image
```

Comment:
how will we know that the image build is passing before merging PR?

Reply:
It will be tested in https://github.com/Mellanox/kubernetes-ci/pull/131